package.json

```diff
@@ -38,7 +38,7 @@
     "tslint-language-service": "^0.9.9",
     "typedoc": "^0.15.0-0",
     "typedoc-plugin-external-module-map": "^1.0.0",
-    "typescript": "^3.1.3",
+    "typescript": "~3.5.3",
```

Comment:
Would it be possible to go for typescript `^3.7.2`? There are very cool features now available: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining 😍

Reply:
Ahah, I know right. But there was [some issue](https://github.com/microsoft/TypeScript/issues/33562) with our `MaybePromise` types... But I can try again :)

Reply:
Update: Still breaks.